<?xml version="1.0.1"?>
<launch>
  <node name="forklift_drive_node" pkg="forklift_drive" type="forklift_drive" output="screen">
    <!-- 串口配置参数 -->
    <param name="port" value="/dev/ttyUSB0" type="str" />
    <param name="baudrate" value="115200" type="int" />
    <param name="loopRate" value="40" type="int" />
  </node>
</launch>
